### PR TITLE
Add Hyundai Veloster 2019

### PR DESCRIPTION
**Pull request by xps-genesis**
Added Hyundai Veloster FP1 and FP2

### CODE REVIEW

**Comment by xps-genesis** [comment]
@CruiseBrantley

**Comment by xps-genesis** [comment]
d624b3d19adce635|2020-07-31--22-39-17

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -138,6 +139,9 @@ class Buttons:
   CAR.PALISADE: [{
     67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 356: 4, 544: 8, 549: 8, 576: 8, 593: 8, 608: 8, 688: 6, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 903: 8, 905: 8, 909: 8, 913: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1064: 8, 1078: 4, 1107: 5, 1123: 8, 1136: 8, 1151: 6, 1155: 8, 1156: 8, 1157: 4, 1162: 8, 1164: 8, 1168: 7, 1170: 8, 1173: 8, 1180: 8, 1186: 2, 1191: 2, 1193: 8, 1210: 8, 1225: 8, 1227: 8, 1265: 4, 1280: 8, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1371: 8, 1378: 8, 1384: 8, 1407: 8, 1419: 8, 1427: 6, 1456: 4, 1470: 8, 2000: 8, 2004: 8, 2005: 8, 2008: 8, 2012: 8
   }],
+  CAR.VELOSTER: [{
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:142]: add `CAR.VELOSTER` to the ignored fingerprints

**Comment by adeebshihadeh** [comment]
Looks like this should be using the `hyundaiLegacy` safety mode

**Comment by xps-genesis** [comment]
there were no errors on the vehicle without legacy check. the test failed even after putting it inl
 legacy check

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (xps-genesis) [reply selfdrive/car/hyundai/values.py:142]: why?

Comment (adeebshihadeh) [reply selfdrive/car/hyundai/values.py:142]: new car ports should only use FW fingerprinting

Comment (xps-genesis) [reply selfdrive/car/hyundai/values.py:142]: so if i add veloster to ignored fingerprint will it still use the FW fobgerprint?

**Comment by adeebshihadeh** [comment]
The panda safety test is passing. The can valid check is the one that’s failing, which means one of the checks [here](https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/carstate.py#L136) is wrong for this car model. It could be the frequency, but it’s more likely checking a message that doesn’t exist on this model.

**Comment by adeebshihadeh** [comment]
Can you replace the route with a longer one with openpilot engaged for at least a minute?

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/hyundai/values.py:142]: yes

**Comment by adeebshihadeh** [comment]
Thanks for the car port, @xps-genesis!

**Pull request merged**